prow/gerrit/adapter/adapter.go

```diff
@@ -233,18 +232,25 @@ func (c *Controller) syncChange(latest client.LastSyncState, changeChan <-chan C
 // Sync looks for newly made gerrit changes
 // and creates prowjobs according to specs
 func (c *Controller) Sync() {
-	processSingleInstance := func(instance string) {
+	processSingleProject := func(instance, project string) {
 		// Assumes the passed in instance was already normalized with https:// prefix.
-		log := logrus.WithField("host", instance)
-		syncTime := c.tracker.Current()
-		latest := syncTime.DeepCopy()
+		log := logrus.WithFields(logrus.Fields{"host": instance, "repo": project})
```

Comment:
It's a little bit unfortunate that we chose the variable names "instance" instead of "host" and "project" instead of "repo". Oh well.

Reply:
Yeah its a bit annoying. We're matching Gerrit's terminology with our variable names, but the logging uses GH terminology so that we can use the same log filters for both.

Reply:
I guess if we were fully using GH terminology `host` would be `org`... Whatever, not our biggest concern right now.